lib/mail/utilities.rb

```diff
@@ -224,7 +225,7 @@ def map_with_index( enum, &block )
       end
 
     end
-    
+
```

Comment:
Yep yep. These are OK, but they're unrelated to the PR so they add a bit of noise for review and to the git history, where a `git blame` will show this commit that has nothing to do with the subject code.